packages/server/src/main/resources/application.properties

```diff
@@ -66,6 +66,7 @@ skema-rs.url=https://skema-rs.staging.terarium.ai
 skema-unified.url=https://api.askem.lum.ai
 terarium.dataservice.url=https://data-service.staging.terarium.ai
 xdd-dev-service.url=https://xdddev.chtc.io
+funman-service.url=http://funman.staging.terarium.ai/api
```

Comment:
Just to confirm is there a corresponding addition to the orchestration project adding this for staging/prod?
Also @dgauldie another entry for local development here